db/table_cache.cc

```diff
@@ -365,17 +383,7 @@ void TableCache::CreateRowCacheKeyPrefix(const ReadOptions& options,
   // the same as there is no snapshot. The exception is that if
   // a seq-checking callback is registered, some internal keys
   // may still be filtered out.
-  uint64_t seq_no = 0;
```

Comment:
Would you mind moving the comment above into the new `GetSequenceNumber` function too?